collect_app/src/test/java/org/odk/collect/android/utilities/MainActivityTest.java

```diff
@@ -0,0 +1,165 @@
+package org.odk.collect.android.utilities;
```

Comment:
Yes @lognaturel. You are right. Package structure of actual classes and their tests should match.